ssawg_trending_scraper.py

```diff
@@ -174,7 +174,7 @@ def get_url(self):
                 stop_minus_start = (Chandra.Time.date2secs(stop_time)
                                     - Chandra.Time.date2secs(start_time))
                 # define halfway through the quarter
-                halfway = Chandra.Time.date2secs(start_time) + stop_minus_start
+                halfway = Chandra.Time.date2secs(start_time) + (stop_minus_start / 2)
```

Comment:
Makes sense and looks good!  Couple style things:
  - If you like `date2secs`, you should probably just import so you don't need the long form every time.
  - Seems like you might be able to skip directly to finding `halfway` from stop_time and start_time as:
      - `halfway = (date2secs(start_time) + date2secs(stop_time)) / 2`
      - (might be able to skip defining `halfway` altogether too and just use it in the if statement but that can start to look ugly if you are also getting now there via datetime etc).
  - From active discussions about time, you could look at astropy.Time or CxoTime and skip most calls to datetime, but that's out of scope of the PR.

Thanks!